src/encoded/visualization.py

```diff
@@ -732,12 +732,11 @@ def add_2d_file_to_higlass_viewconf(views, new_file):
 def get_title(file):
     """ Returns a string containing the title for the view.
     """
-    title = file["display_title"]
-    if "file_classification" in file:
-        title += "({classification})".format(
-            classification = file["file_classification"]
-        )
-    return title
+    # Use the track title. As a fallback, use the display title.
+    if "track_title" in file:
```

Comment:
In case `track_title` gets a value of None, this should probably be either:
`if "track_title"` in file and file["track_title"]:`
or equivalently
`if file.get("track_title"):`